# point_sim_controller.py and point_planner.py config file

sim:
  update_freq: 50
  random_pose_min_max: [-100, 100]
  random_yaw: false
  
planners:
  update_freq: 50
  random_twist_linear_min_max: [-10,10]
  random_twist_angular_min_max: [-0.0,0.0]
  
actors_list:
  - kipp
  - tars
  - case
  - deckard
  - hal
  - ava

kipp:
  planner: linear
  start_pos:        # either x,y,z,psi or 'random'
    x: 5
    y: 4
    z: 3
    psi: 3.14
  start_twist:
    x: 1
    y: 1
    z: -1
    psi: 0

tars:
  planner: linear
  start_pos: random
  start_twist: random

case:
  planner: linear
  start_pos: random
  start_twist: random

deckard:
  planner: linear
  start_pos: random
  start_twist: random

hal:
  planner: linear
  start_pos: random
  start_twist: random

ava:
  planner: linear
  start_pos: random
  start_twist: random
